source/Stargate-Model/ResourceNotPaginatedPolicy.class.st

```diff
@@ -0,0 +1,36 @@
+Class {
+	#name : #ResourceNotPaginatedPolicy,
```

Comment:
It should be ResourceIsNotPaginatedPolicy. I'm not sure with the use of the resource as singular.

Maybe PaginationPolicy with two subclasses:
 - DoNotPaginatePolicy 
 - PaginatePolicy